default.yml

```diff
@@ -41,6 +41,14 @@ Metrics/BlockLength:
     # Specs deliberately will almost always break this cop (It should be added to rubocop-rspec defaults ideally)
     - spec/**/*
 
+# Count arrays and hashes as one-line when calculating method length
+Metrics/MethodLength:
+  CountAsOne: ['array', 'hash']
```

Comment:
I think in the next 12/18 months this will become the defacto norm. I'm seeing this config in so many places for things